Backend/Remora.Discord.API.Abstractions/API/Objects/Users/IUser.cs

```diff
@@ -55,7 +55,11 @@ public interface IUser : IPartialUser
     /// <summary>
     /// Gets the user's display name, if it is set. For bots, this is the application name.
     /// </summary>
-    new string? GlobalName { get; }
+    /// <remarks>
+    /// Intentionally not implemented as this field is not present in Discord's OpenAPI specification.
```

Comment:
This, for example, is a good text you can move there instead.

Reply:
Put this on the remarks for IUser and used inheritdoc to copy just that remark to the other classes/interfaces I removed global_name from